components/match2/wikis/rocketleague/match_group_input_custom.lua

```diff
@@ -372,6 +381,12 @@ end
 -- map related functions
 --
 function mapFunctions.getExtraData(map)
+	local timeoutInput = mw.text.split(map.timeout or '', ',')
+	local timeouts = {}
+	for _, timeoutValue in pairs(timeoutInput) do
+		table.insert(timeouts, tonumber(timeoutValue))
+	end
+
```

Comment:
Is it not possible to do something like 
```suggestion
	local timeoutInput = Table.map(mw.text.split(map.timeout or '', ','), (t) return tonumber(timeoutValue) end)
```

Reply:
```lua
	local timeouts = Table.mapValues(mw.text.split(map.timeout or '', ','), tonumber)
```
and changing  `#timeouts > 0` to `Table.isNotEmpty(timeouts)` should do it.

Reply:
would still leave empty holes in the stored data, right?
like `|timeout=,1` would still be stored as
`{2 = 1}` instead of `{1}` (and for `|timeout=1,,2` we get `{1, 3 = 2}` instead of `{1, 2}`)

if that is okay data wise / we assume such entries do not happen we can use `Table.mapValues` for sure

Reply:
Could do an `Array.extractValues()` after the mapValues, then there won't be any gaps